justfile

```diff
@@ -0,0 +1,14 @@
+clean:
+    -rm -rf /Users/agelameris/.g8/boot/org.foundweekends.giter8/giter8_2.12/0.13.1-SNAPSHOT
+    -rm -rf /Users/agelameris/.ivy2/local/org.foundweekends.giter8/sbt-giter8/scala_2.12/sbt_1.0/0.13.1-SNAPSHOT/
```

Comment:
Was this file meant to be included into the PR?

Reply:
No, it's my bad, sorry. I'll delete it in another PR

Reply:
I had to push force ofc, sorry for history rewriting